app/view_models/insights_view_model.rb

```diff
@@ -63,7 +139,9 @@ def accepted_auctions_count
     accepted_auctions.count
   end
 
-  private
+  def accepted_auction_percent
```

Comment:
So, this will dip when an auction is running or even when an auction is over but the delivery deadline has not been met. I wonder if we should change the denominator to be simply the total of accepted + rejected?